extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -75,5 +72,15 @@ describes.realWin(
       expect(shoppingTag.element.textContent).to.be.empty;
       expect(shoppingTag.isLayoutSupported(Layout_Enum.CONTAINER)).to.be.true;
     });
+
+    it('should set active product in store service when shopping tag is clicked', async () => {
+      shoppingTag.element.setAttribute('data-tag-id', 'sunglasses');
+      await shoppingDataDispatchStoreService();
+      env.sandbox.stub(shoppingTag, 'mutateElement').callsFake(() => {
```

Comment:
What is `mutateElement` used for here? We're not updating the dom. 🤔

Reply:
We are calling `renderShoppingTagTemplate`, which adds the onClick event - this is only called inside mutateElement (see the call to `renderShoppingTagTemplate` in `amp-story-shopping-tag.js`)

Reply:
Got it! Thanks for explaining :)